README.md

```diff
@@ -1 +1,66 @@
-# First-capstone-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Marvel Divide 2015
+
+![Marvel Divide 2015 logo](./Images/logo/marvel-logo_250x150.gif)
+
+> This is an educational project to make a website for a conference event with home page and about page.
+
+## Built With
+
+- HTML5, CSS3, JavaScript
+- Linters:
+  - Stylelint
+  - Lighthouse
+  - Webhint
+  - ESLint
+
+## Project Documentation
+
+- Here is the [link]() to my loom video
```

Comment:
- [ ] [REQUIRED] You should record a `video`(`Video presentation`) presenting the features of the project!

![image](https://user-images.githubusercontent.com/63915024/192010458-eddb1203-61f7-4d04-af9d-f34476f15ebc.png)

Reply:
good  looking out